testing/trino-testing/src/main/java/io/trino/testing/AbstractTestDistributedQueries.java

```diff
@@ -72,14 +72,9 @@
         extends AbstractTestQueries
 {
     protected boolean supportsCreateSchema()
```

Comment:
If you're removing one of these then rename the method too.

But I would actually prefer keeping the `supportsDropSchema` and using that in the drop related method.

Either would work.